contracts/src/interfaces/IMVPSybil.sol

```diff
@@ -3,29 +3,22 @@ pragma solidity 0.8.23;
 
 interface IMVPSybil {
     error InvalidVerifierAddress();
-    error InvalidCreateAccountTransaction();
-    error InvalidDepositTransaction();
-    error InvalidForceExitTransaction();
-    error InvalidForceExplodeTransaction();
-    error InternalTxNotAllowed();
     error BatchTimeoutExceeded();
-    error LoadAmountExceedsLimit();
     error LoadAmountDoesNotMatch();
     error AmountExceedsLimit();
-    error InvalidTransactionParameters();
     error WithdrawAlreadyDone();
     error SmtProofInvalid();
     error EthTransferFailed();
     error InvalidProof();
-
-
+    error InvalidFromIdx();
+    error InvalidToIdx();
 
     // Initialization function
     function initialize(
         address[] memory verifiers,
         uint256[] memory maxTxs,
         uint256[] memory nLevels,
-        uint8 _forgeL1L2BatchTimeout, 
+        uint8 _forgeL1L2BatchTimeout,
```

Comment:
It should be L1 only or remove both as we are only dealing with L1

Reply:
Okay, I will do that.